engine/crates/graphql-schema-diff/tests/diff/field_wrapping_type_changes.graphql

```diff
@@ -0,0 +1,13 @@
+scalar Date
```

Comment:
This wasn't tested directly in this crate, but in operation-checks. Added this test for tighter feedback loop when debugging.